static/views/landing.html

```diff
@@ -1376,6 +1383,26 @@ <h4 style="margin-left: 9px;">{{ .SiteName }}</h4>
 				jobNode.setAttribute('data-toggle', 'off');
 			}
 		}
+		function saveJob(jobID, userID) {
+			var formData = new FormData();
+			formData.append('jobID', jobID);
+			formData.append('developerID', userID);
+
+			post('x/s/e', formData, () => {
+				if (status == 200) {
```

Comment:
switch case is probably more appropriate here, or just like we do in other cases, if 200 saved successfully else there was an error